datafusion/expr/src/built_in_function.rs

```diff
@@ -1186,7 +1194,9 @@ impl BuiltinScalarFunction {
             | BuiltinScalarFunction::Sqrt
             | BuiltinScalarFunction::Tan
             | BuiltinScalarFunction::Tanh
-            | BuiltinScalarFunction::Cot => {
+            | BuiltinScalarFunction::Cot
+            | BuiltinScalarFunction::Isnan
+            | BuiltinScalarFunction::Iszero => {
```

Comment:
I would consider `Isnan` and `Iszero` not math expressions as listed here. The reason to give f64 high priority is also not applied too for these two expressions, I think.

Reply:
Based on above reason, maybe it is less confused to have them in a separate pattern.

Reply:
Sorry for the late reply.
Yes, f64 doesn't need high priority.
I've changed it.